core/backend/src/rpc/web/response.ts

```diff
@@ -24,6 +24,10 @@ import {
   WebAppRpcRequest,
 } from "@itwin/core-common";
 
+import { Readable, Stream } from "stream";
```

Comment:
could you prefix all node api imports with the node protocol?

eg

```suggestion
import { Readable, Stream } from "node:stream";
```

Reply:
Prefixed all new Node imports in this PR